src/assets/styles/abstracts/_variables.css

```diff
@@ -0,0 +1,20 @@
+:root {
+    /** Typography **/
+    --family-sans-serif: "Source Sans 3", sans-serif;
+    --font-weight-normal: 400;
+    --font-weight-medium: 500;
+    --font-weight-semibold: 600;
+    --font-weight-bold: 700;
+
+    /** Colours **/
+
+    --blue: hsla(194, 60%, 59%, 1);
```

Comment:
I'd prefix colours with `--color-` for consistency. It's helpful for grouping variables.